tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -0,0 +1,268 @@
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" copies file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Alice" the content of the file "testshare.txt" of the space "Project" should be "testshare content"
+    Examples:
+      | role    | permissions |
+      | manager | 31          |
+      | manager | 17          |
+      | editor  | 31          |
+      | editor  | 17          |
+
+
+  Scenario Outline: User copy a file from space shares jail with different role to space project with role viewer
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has shared a space "Project" to user "Alice" with role "viewer"
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" copies file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "403"
+    Examples:
+      | permissions |
+      | 31          |
+      | 17          |
+
+
+  Scenario Outline: User copy a file from space shares jail with different role to space shares jail with role editor
+    Given user "Brian" has created folder "/testshare1"
+    And user "Brian" has created folder "/testshare2"
+    And user "Brian" has uploaded file with content "testshare1 content" to "/testshare1/testshare1.txt"
+    And user "Brian" has shared folder "/testshare1" with user "Alice" with permissions "<permissions>"
+    And user "Brian" has shared folder "/testshare2" with user "Alice" with permissions "31"
+    And user "Alice" has accepted share "/testshare1" offered by user "Brian"
+    And user "Alice" has accepted share "/testshare2" offered by user "Brian"
+    When user "Alice" copies file "/testshare1/testshare1.txt" from space "Shares Jail" to "/testshare2/testshare1.txt" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And as "Alice" file "/Shares/testshare2/testshare1.txt" should exist
+    And as "Brian" file "/testshare2/testshare1.txt" should exist
+    And the content of file "/Shares/testshare2/testshare1.txt" for user "Alice" should be "testshare1 content"
+    And the content of file "/testshare2/testshare1.txt" for user "Brian" should be "testshare1 content"
```

Comment:
@ScharfViktor we might need to implement all these steps for space `Project` and `Shares Jail` in ocis.